www/front_src/src/Resources/Listing/useLoadResources/index.ts

```diff
@@ -65,14 +67,20 @@ const useLoadResources = (): LoadResources => {
       search,
     }).then(setListing);
 
+    if (isNil(details)) {
+      return;
+    }
+
     loadDetails();
   };
 
   const initAutorefresh = (): void => {
     window.clearInterval(refreshIntervalRef.current);
 
     const interval = enabledAutorefresh
-      ? window.setInterval(load, refreshIntervalMs)
+      ? window.setInterval(() => {
+          load();
```

Comment:
The old implementation has the same behavior, right ?

Reply:
yes